modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -78,6 +96,7 @@ const cellOperations = function (mutate: (e1: SugarElement, e2: SugarElement) =>
 
     cloneCss(prev, td);
     mutate(prev.element, td);
+    transferAppropriateAttributes(prev.element.dom, td.dom);
```

Comment:
This should be before the mutate line, incase the mutate function needs to make use of it.